src/test/java/integration/CounterpartyControllerIT.java

```diff
@@ -123,15 +126,15 @@ public void createCounterparty() throws Exception {
                         statusCode(SC_OK).
                 extract()
                         .response();
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
         //check created data
         Counterparty createdCounterparty = counterpartyService
                 .getEntityByUuidAnonymous(UUID.fromString(response.path("uuid")));
-        long timeCreated = createdCounterparty.getCreated().getTime();
-        long timeModified = createdCounterparty.getLastModified().getTime();
+        LocalDateTime timeCreated = createdCounterparty.getCreated();
+        LocalDateTime timeModified = createdCounterparty.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).